pkgs/development/interpreters/elixir/1.14.nix

```diff
@@ -0,0 +1,10 @@
+{ mkDerivation }:
+
+# How to obtain `sha256`:
+# nix-prefetch-url --unpack https://github.com/elixir-lang/elixir/archive/v${version}.tar.gz
```

Comment:
```suggestion
```
This is already in the contributing guide

Reply:
Thanks for the review! Removed in the version committed as b73e0e96e3bf9644dbbf1bb4bc26aa2c87cf50e0

Reply:
> PS: There are probably more of this comments across the code base from me - should I remove them all?

If the download link is not obvious then we can keep it but for github downloads at least we do not require them.